incubator-tools/language_detection/language_detection.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        parent = f"projects/{project_id}/locations/{location}"
</pre><p>There should be a way to form this using a client method.</p><p> </p><p><code>client.project_location_path()</code> or something like it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Marking as done. Replaced with <code>parent=client.common_location_path(project=project_id, location=location)</code></p>